src/raftstore/store/peer.rs

```diff
@@ -330,19 +331,29 @@ impl Peer {
     fn send_ready_metric(&self, ready: &Ready) {
         if !ready.messages.is_empty() {
             metric_count!("raftstore.send_raft_message", ready.messages.len() as i64);
+            PEER_RAFT_READY_COUNTER_VEC.with_label_values(&["send raft message"])
```

Comment:
seem a little long for the label value
maybe we can use message, committed, append and snapshot directly?